ununifi/chain.json

```diff
@@ -0,0 +1,174 @@
+{
+  "$schema": "../chain.schema.json",
+  "chain_name": "ununifi",
+  "status": "live",
+  "network_type": "mainnet",
+  "website": "https://ununifi.io/",
+  "pretty_name": "UnUniFi",
+  "chain_id": "ununifi-beta-v1",
+  "bech32_prefix": "ununifi",
```

Comment:
Please register this prefix: https://github.com/satoshilabs/slips/blob/master/slip-0173.md

Reply:
Now, PR created and awaiting merge.

Reply:
merged